tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,312 @@
+})
+    
+db, err := sql.Open("mysql", "root:<your_password>@tcp(tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com:4000)/test?tls=tidb")
+```
+
+You can click **show example usage** to view detailed code examples.
+
+```
+package main
+import (
+  "crypto/tls"
+  "crypto/x509"
+  "database/sql"
+  "fmt"
+  "io/ioutil"
+  "log"
+  
+  "github.com/go-sql-driver/mysql"
+)    
+func main() {
+  rootCertPool := x509.NewCertPool()
+  pem, err := ioutil.ReadFile("ca.pem")
+  if err != nil {
+    log.Fatal(err)
+  }
+  if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
+    log.Fatal("Failed to append PEM.")
+  }
+  mysql.RegisterTLSConfig("tidb", &tls.Config{
+    RootCAs:    rootCertPool,
+    MinVersion: tls.VersionTLS12,
+    ServerName: "tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com",
+  })
+  db, err := sql.Open("mysql", "root:<your_password>@tcp(tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com:4000)/test?tls=tidb")
+  if err != nil {
+    log.Fatal("failed to connect database", err)
+  }
+  defer db.Close()
+
+  var dbName string
+  err = db.QueryRow("SELECT DATABASE();").Scan(&dbName)
+  if err != nil {
+    log.Fatal("failed to execute query", err)
+  }
+  fmt.Println(dbName)
+}
+```
+
+Parameter description：
+
+- Register `tls.Config` in connection to enable TLS and validate TiDB Dedicated Tier clusters. 
+- Set `MinVersion: tls.VersionTLS12` to restrict the versions of TLS protocol.
+- Set `ServerName: "<host>"` to verify TiDB Dedicated Tier's hostname.
+- If you do not want to register a new TLS configuration, you can just set `tls=true` in the connection string.
+
+</div>
+
+<div label="Node.js">
+
+[Mysql](https://www.npmjs.com/package/mysql2)'s TLS connection configurations are used here as an example.
+
+```
+var connection = mysql.createConnection({
+  host: 'tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com',
+  port: 4000,
+  user: 'root',
+  password: '<your_password>',
+  database: 'test',
+  ssl: {
+    ca: fs.readFileSync('ca.pem'),
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true
+  }
+});
+```
+
+You can click **show example usage** to view detailed code examples.
+
+```
+var mysql = require('mysql2');
+var fs = require('fs');
+var connection = mysql.createConnection({
+  host: 'tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com',
+  port: 4000,
+  user: 'root',
+  password: '<your_password>',
+  database: 'test',
+  ssl: {
+    ca: fs.readFileSync('ca.pem'),
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true
+  }
+});
+connection.connect(function(err) {
+  if (err) {
+    throw err
+  }
+  connection.query('SELECT DATABASE();', function(err, rows) {
+    if (err) {
+      throw err
+    }
+    console.log(rows[0]['DATABASE()']);
+    connection.end()
+  });
+});
+```
+
+Parameter description：
+
+- Set `ssl: {minVersion: 'TLSv1.2'}` to restrict the versions of TLS protocol.
+- Set `ssl: {ca: fs.readFileSync('ca.pem')}` to read the special path ca.pem.
```

Comment:
```suggestion
- Set `ssl: {ca: fs.readFileSync('<CA_root_path>')}` to read the content of cert.
```

Reply:
Does it mean "- Update `ca.pem` in `ssl: {ca: fs.readFileSync('ca.pem')}` to your local path of the downloaded TiDB cluster CA."